### PR TITLE
Rewrite and update boilerplate to work in 2022.

### PR DESCRIPTION
**Pull request by Naquino14**
This PR keeps most of the same functionality and look from the original boilerplate. The main purpose was to update all the packages to work in 2022.

### CODE REVIEW

**Comment by Naquino14** [comment]
I realized I needed to change the README to fit the new boilerplate. Im also thinking of making changes to the dockerfile.

File: .env [hunk 1]
```diff
@@ -1,2 +1,4 @@
-REACT_APP_SSO_CLIENT_ID=react-boilerplate
-REACT_APP_SSO_AUTHORITY=https://sso.csh.rit.edu/auth/realms/csh
\ No newline at end of file
+REACT_APP_SSO_CLIENT_ID=imhere2
```
Comment (devinmatte) [inline .env:1]: Is this meant to be changed?

File: public/manifest.json [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
+      "src": "favicon.svg",
+      "sizes": "64x64 32x32 24x24 16x16",
+      "type": "image/x-icon"
+    }
+  ],
+  "start_url": ".",
   "display": "standalone",
-  "lang": "en",
-  "orientation": "portrait",
-  "theme_color": "#b0197e",
```
Comment (devinmatte) [inline public/manifest.json:9]: Some of these might be nice defaults to keep

**Review by devinmatte — APPROVED** [review]
If it works for people, go for it. I left two comments that you can take or leave. This project is yours now, hopefully it helps people build new apps

File: public/manifest.json (hunk 2 above)
Comment (Naquino14) [reply public/manifest.json:9]: I might have deleted these accidentally. Ill put it back later today.

**Review by Naquino14 — COMMENTED** [review]
👍

**Pull request merged**